linodecli/configuration/auth.py

```diff
@@ -340,11 +341,12 @@ def log_message(self, form, *args):  # pylint: disable=arguments-differ
             # serve requests one at a time until we get a token or are interrupted
             serv.handle_request()
     except KeyboardInterrupt:
-        print()
+        print(file=sys.stderr)
         print(
             "Giving up.  If you couldn't get web authentication to work, please "
             "try token using a token by invoking with `linode-cli configure --token`, "
-            "and open an issue at https://github.com/linode/linode-cli"
+            "and open an issue at https://github.com/linode/linode-cli",
+            file=sys.stderr
```

Comment:
```suggestion
        print(
            "\nGiving up.  If you couldn't get web authentication to work, please "
            "try token using a token by invoking with `linode-cli configure --token`, "
            "and open an issue at https://github.com/linode/linode-cli"
            "and open an issue at https://github.com/linode/linode-cli",
            file=sys.stderr
```
Let's merge these 2 print calls?

Reply:
Done.